core/src/main/java/nl/nn/adapterframework/unmanaged/SpringJmsConnector.java

```diff
@@ -216,6 +216,7 @@ public void start() throws ListenerException {
 			try {
 				jmsContainer.start();
 				if (pollGuardInterval != -1 && jmsContainer instanceof IbisMessageListenerContainer) {
+					log.debug("Creating poll-guard timer with interval [{}ms] while starting SpringJmsConnector", pollGuardInterval);
```

Comment:
Ik ben bang dat dit wel heel veel log regels op gaat leveren wanneer men 30+ jms listeners heeft. Is het niet handig deze naar trace te verplaatsen?

Reply:
Hm ik snap het probleem. Ik zou eigenlijk in 7.9 de PollGuard anders willen implementeren zodat deze maar 1x wordt aangemaakt en intern de timer afhandelt zoals de TimeoutGuard dat doet, alleen dat was een grotere aanpassing dan ik wou doen in 7.7 voor alleen het toevoegen van logging.